framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.11 under development
 ------------------------
 
+- Enh #12798: Changed `yii\cache\Dependency::getHasChanged()` (to be deprecated in 2.1) to `yii\cache\Dependency::isChanged()` (dynasource)
```

Comment:
It still doesn't make any sense. It **is** deprecated in 2.0.11, not in 2.1. Can't we just change this to something like "Replace `yii\cache\Dependency::getHasChanged()`  by `yii\cache\Dependency::isChanged()`"?